api/v1/reviewers/suggestions/formRequests/AddReviewerSuggestion.php

```diff
@@ -0,0 +1,99 @@
+<?php
+
+namespace PKP\API\v1\reviewers\suggestions\formRequests;
+
+use APP\core\Application;
+use Illuminate\Foundation\Http\FormRequest;
+use Illuminate\Validation\Rule;
+
+class AddReviewerSuggestion extends FormRequest
+{
+    /**
+     * @copydoc \Illuminate\Foundation\Http\FormRequest::validated()
+     */
+    public function validated($key = null, $default = null)
+    {
+        $validated = parent::validated($key, $default);
+
+        return collect($validated)->map(
+            fn($value) => is_array($value) ? array_filter($value) : $value
+        )->toArray();
+    }
+
+    /**
+     * Get the validation rules that apply to the request.
+     */
+    public function rules(): array
+    {
+        return [
+            'submissionId' => [
+                'required',
+                'integer',
+                Rule::exists('submissions', 'submission_id'),
+            ],
+            'suggestingUserId' => [
+                'sometimes',
+                'nullable',
+                'integer',
+                Rule::exists('users', 'user_id'),
+            ],
+            'familyName' => [
+                'required',
+                'array',
+            ],
+            'givenName' => [
+                'required',
+                'array',
+            ],
+            'email' => [
+                'required',
+                'email',
+            ],
+            'affiliation' => [
+                'required',
+                'array',
+            ],
+            'suggestionReason' => [
+                'required',
+                'array',
+            ],
+        ];
+    }
+
+    /**
+     * Get the error messages for the defined validation rules.
+     *
+     * @return array<string, string>
+     */
+    public function messages(): array
+    {
+        return [
+            'familyName.required' => 'family name is required',
+        ];
+    }
```

Comment:
There are placeholder for now as part of experiment but this is what needed to be resolved as dynamic multilingual response for error .